src/Tools/dotnet-monitor/Strings.resx

```diff
@@ -565,6 +565,9 @@
     <comment>Gets the format string that is printed in the 14:InsecureAutheticationConfiguration event.
 0 Format Parameters</comment>
   </data>
+  <data name="LogFormatString_IntermediateFileDeletionFailed" xml:space="preserve">
```

Comment:
All of the format strings that are used for logging should use named holes rather than indexed holes: e.g. change `{0}` to `{path}`.